website/pages/en/index.js

```diff
@@ -96,7 +96,7 @@ const TldrSection = ({ language }) => (
         <div style={{ display: "flex", flexDirection: "column" }}>
           <h2>Why?</h2>
           <ul style={{ flex: "1" }}>
-            <li>You press save and code is formatted</li>
+            <li>Click save and your code is formatted</li>
```

Comment:
The [Microsoft Style guide](https://learn.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui) has some good information about describing interactions with UI:

> Customers interact with products using different input methods: keyboard, mouse, touch, voice, and more. So use generic verbs that work with any input method. Don't use input-specific verbs, such as _click_ or _swipe_. Instead, use the verbs listed here.

The document listed above also has a table of good verbs, but none of these fit the Prettier homepage... 😞 

Based on their recommendation, my suggestion to use `click` doesn't fix the problem. 😄 So how about using `save` as the verb? 

```suggestion
            <li>Formats code when saving files</li>
```